packages/web-twig/src/Resources/components/Accordion/AccordionHeader.twig

```diff
@@ -0,0 +1,50 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _class = props.class | default(null) -%}
+{%- set _id = props.id -%}
+{%- set _for = props.for -%}
+{%- set _elementType = (props.elementType is defined) ? props.elementType : 'h3' -%}
+{%- set _isOpen = props.isOpen | default(false) | boolprop -%}
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ 'Accordion__itemHeader' -%}
+{%- set _toggleClassName = _spiritClassPrefix ~ 'Accordion__itemToggle' -%}
+{%- set _sideClassName = _spiritClassPrefix ~ 'Accordion__itemSide' -%}
+{%- set _slotClassName = _spiritClassPrefix ~ 'Accordion__itemSlot' -%}
+{%- set _iconClassName = _spiritClassPrefix ~ 'Accordion__itemIcon' -%}
+
+{# Attributes #}
+{%- set _idAttr = _id ? 'id="' ~ _id ~ '"' : null -%}
+{%- set _targetAttr = _for ? 'data-target="' ~ _for ~ '"' : null -%}
```

Comment:
To keep the code consistent (and also avoid confusion with the `target` HTML attribute in this case), this could be better:

```suggestion
{%- set _dataTargetAttr = _for ? 'data-target="' ~ _for ~ '"' : null -%}
```

Reply:
fixed